lib/dirList.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+const path = require('path')
+const fs = require('fs')
+
+const dirList = {
+  /**
+   * get files and dirs from dir, or error
+   * @param {string} dir full path fs dir
+   * @param {function(error, entries)} callback
+   * note: can't use glob because don't get error on non existing dir
+   */
+  list: function (dir, callback) {
```

Comment:
Is `dir` populated from remote user input? If so, is `dir` locked down to a specific root? Can a path like `../../../../../etc` be passed in and result in a directory tree being returned?

Reply:
`dir` comes "send" module from the requested route, so, no it does not expose something behind the "root" doing `curl localhost:3001/../../../index`